packages/backend/server/src/base/helpers/index.ts

```diff
@@ -4,12 +4,13 @@ import { Global, Module } from '@nestjs/common';
 
 import { CryptoHelper } from './crypto';
 import { URLHelper } from './url';
+import { WorkspaceHelper } from './workspace';
 
 @Global()
 @Module({
-  providers: [URLHelper, CryptoHelper],
-  exports: [URLHelper, CryptoHelper],
+  providers: [URLHelper, CryptoHelper, WorkspaceHelper],
```

Comment:
base modules should not get invoived with business logic. put it in `WorkspaceBlobStorage`